src/server/editorServices.ts

```diff
@@ -1536,7 +1549,7 @@ namespace ts.server {
         /*@internal*/
         findDefaultConfiguredProject(info: ScriptInfo) {
             if (!info.isScriptOpen()) return undefined;
-            const configFileName = this.getConfigFileNameForFile(info);
+            const configFileName = this.getConfigFileNameForFile(info, /*stopAtNodeModules*/ false);

```

Comment:
@sheetalkamat Would it be better to just always stop at `node_modules`?  I couldn't figure out whether introducing this inconsistency to maintain back compat would cause problems.

Reply:
I think instead of changing how we look for default config file name, we should change `assignProjectToOpenedScriptInfo` to not create configured project if default configured project is not loaded and script info has `/node_modules/`  in its path and its not orphan

Reply:
I passed false here to retain the old behavior.  `assignProjectToOpenedScriptInfo` passes true, as does `reloadConfiguredProjectForFiles`.

I'm not sure I understand the orphan check in general or how it applies to this case.  I think, roughly, you mean that we might want to do it if the node_modules file isn't in some other project, but I'm not sure I agree - it seems like we never want to load a configured project for a node_modules file.

Reply:
Well if that's the case why look for default configured project for such script infos in first place That is if block completely at  https://github.com/microsoft/TypeScript/blob/master/src/server/editorServices.ts#L2670 completely?

Reply:
The reason I made the change at a lower level than `assignProjectToOpenedScriptInfo` was because I thought other consumers of `getConfigFileNameForFile` might need to be updated as well.  For example, it seems like `reloadConfiguredProjectForFiles` should also not consider the default configured project.

Reply:
@sheetalkamat I don't feel strongly about how this should be implemented.  If you have a different approach in mind, please let me know and I'll do my best to adopt it.